### ISSUE TITLE
Creating local dvid repo and adding dataset: SIGSEGV

### ISSUE THREAD

**Issue by ssinhaleite** [janelia-flyem/dvid#243]
Hello, 
I was trying to create a local DVID server and I followed the steps indicated here:
https://github.com/janelia-flyem/dvid/wiki/Simple-Example#create-a-new-repository-repo

Everything worked fine to create the repo and the datatype. However, when I tried to add the data to the repo, I got a SIGSEGV. And after that, the server can't be initialized anymore.

The terminal where I was using the commands shows everything working fine (but the last command because the server was turned off):
```
vleite@zeitgeist:~/research/dvid/dvid-distro$ ./bin/dvid repos new Dataset1 "Dataset for testing DVID use"
New repo "Dataset1" created with head node 1314ed176f1b4c8c9cc571d439301e87
vleite@zeitgeist:~/research/dvid/dvid-distro$ ./bin/dvid types

Data Types within this DVID Server
----------------------------------
vleite@zeitgeist:~/research/dvid/dvid-distro$ ./bin/dvid repo 13 new uint8blk grayscale
Data "grayscale" [uint8blk] added to node 1314ed176f1b4c8c9cc571d439301e87
vleite@zeitgeist:~/research/dvid/dvid-distro$ ./bin/dvid types

Data Types within this DVID Server
----------------------------------
uint8blk             github.com/janelia-flyem/dvid/datatype/imageblk/uint8.go
vleite@zeitgeist:~/research/dvid/dvid-distro$ ./bin/dvid node 13 grayscale load 100,100,2600 "/home/vleite/research/dvid/sample-grayscale/*.png"
Loading 250 files into data instance "grayscale" @ node 13...
vleite@zeitgeist:~/research/dvid/dvid-distro$ ./bin/dvid types
2018/01/19 10:33:30 gorpc.Client: [localhost:8001]. Cannot establish rpc connection: [dial tcp 127.0.0.1:8001: getsockopt: connection refused]
```

But, the terminal where I started the server disagrees: https://gist.github.com/ssinhaleite/6b39333533fe7e5084d419adf94a27a5

What can I do to add the data to the server properly?
Thanks!

**Comment by DocSavage** [comment]
My apologies for the delay in response.  The issue notification got sent to spam :/
This looks like an issue with the version of Go lz4 compatibility with newer Linux.  I will try to update this if I can replicate on our local server.
https://github.com/janelia-flyem/dvid/issues/163

**Comment by DocSavage** [comment]
Fixed in commit 7380190

**Comment by DocSavage** [comment]
Master branch has reverted to older CGo lz4 due to issues. See issue #251.

**Comment by DocSavage** [comment]
This issue has been solved by updating the go lz4 library with new lz4 source code. See closed issue #251
